assets/svgxml/right-arrow.ts

```diff
@@ -0,0 +1,7 @@
+const rightArrow = `<svg width="19" height="14" viewBox="0 0 19 14" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
to keep up with what you did so far i would exported the SvgXml here instead of exporting the xml